RELEASE-NOTES.md

```diff
@@ -1,7 +1,11 @@
 # Wikibase DataModel release notes
 
-## Version 4.4.0 (dev)
-
+## Version 5 (dev)
```

Comment:
No need to make breaking changes. `TermFallback` and `DerivedPropertyValueSnak` can be deprecated and scheduled for removal in the next breaking release.